pkg/server/proto/server.proto

```diff
@@ -4728,6 +4734,28 @@ message Pipeline {
   }
 }
 
+message PipelineRun {
+  string id = 1;
+  // The sequence number for this pipeline run.
+  uint64 sequence = 2;
+
+  Ref.Pipeline pipeline = 3;
+
+  Ref.Job root_job = 4;
+  Ref.Job last_job = 5;
```

Comment:
Thinking about this after we paired, maybe it makes sense to have a map of step names to job refs? Then we can see the status of every step job for a given pipeline run.